### PR TITLE
Css helpers shorthand implementation

### PR DESCRIPTION
**Pull request by ulivz**
# Summary 

Finished https://github.com/egoist/docute/issues/145, two main points:

- Add fenced container syntax for css helper by using [markdown-it-container](https://github.com/markdown-it/markdown-it-container)
- Migrate the style from v3.

# Snapshot

image

### CODE REVIEW

File: packages/docute/src/utils/markdown-it/cssHelpers.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    validate (params) {
+      return params.trim().match(RE)
+    },
+
+    render(tokens, idx) {
+      const [, helperType] = tokens[idx].info.trim().match(RE) || []
+      // TODO Temporarily using div instead of p is because a bug of markdown-it-container
+      // Refer to https://github.com/markdown-it/markdown-it-container/issues/22
+      if (tokens[idx].nesting === 1) {
+        return `<div class="${helperType}">\n` // opening tag
```
Comment (ulivz) [inline packages/docute/src/utils/markdown-it/cssHelpers.js]: Rendering this class can be replaced by `md.renderer.renderAttr`, but this style looks more `Intuitive`

**Comment by ulivz** [comment]
@egoist Just hold on this PR, thanks. I have made a standalone test for using `<p class='xxx'>` instead of `<div>`, and it looks good. I need to do some investigation for that.

https://codepan.net/gist/0d29a35d61822f769c73351582a65b99

**Comment by ulivz** [comment]
@egoist It seems that this PR can be reviewed now.

**Comment by egoist** [comment]
Deploy preview for *docute* ready!

Built with commit b44a0aa3c1996028c88cd6c7214cd5c991fa1750

https://deploy-preview-152--docute.netlify.com

**Comment by ulivz** [comment]
I have merged `special-blockquotes` into this branch, so this branch shouldn't be merge util its dependence merged/

**Comment by egoist** [comment]
seems we can simply convert `::: warning` `::: info` `::: alert` `::: success` to `<div class="message $type">...</div>` to reuse styles from #157

**Comment by ulivz** [comment]
Not sure if the naming of message container or colors is OK for you. so this is up to you to decide.

Weekend is done, Oh my gad.

**Comment by ulivz** [comment]
@egoist Cool, just let me to work for it.

**Comment by ulivz** [comment]
@egoist Quick question before do some change, my concern is that if user want to write some multiple-line syntax, such as `table`, which will be unable to do since `::: $type` is only a single line syntax.

**Pull request closed without merging**